hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/TestLatencyTracker.java

```diff
@@ -0,0 +1,479 @@
+      latencyTracker.recordClientLatency(Instant.now(), null, null, false, null);
+      latencyTracker.recordClientLatency(Instant.now(), "test", null, false, null);
+      latencyTracker.recordClientLatency(Instant.now(), "test", "test", false, null);
+      latencyTracker.recordClientLatency(Instant.now(), "test", "test", false, httpOperation);
+
+      latencyTracker.recordClientLatency(null, null, null, null, false, null);
+      latencyTracker.recordClientLatency(Instant.now(), null, null, null, false, null);
+      latencyTracker.recordClientLatency(Instant.now(), Instant.now(), null, null, false, null);
+      latencyTracker.recordClientLatency(Instant.now(), Instant.now(), "test", null, false, null);
+      latencyTracker.recordClientLatency(Instant.now(), Instant.now(), "test", "test", false, null);
+      latencyTracker.recordClientLatency(Instant.now(), Instant.now(), "test", "test", false, httpOperation);
+
+      latencyTracker.recordClientLatency(testInstant, Instant.now(), null, null, false, null);
+      latencyTracker.recordClientLatency(Instant.MAX, Instant.now(), null, null, false, null);
+      latencyTracker.recordClientLatency(Instant.now(), Instant.MIN, null, null, false, null);
+
+      latencyTracker.recordClientLatency(null, null, null, false, null, 0, null);
+      latencyTracker.recordClientLatency(Instant.now(), null, null, false, null, 0, null);
+      latencyTracker.recordClientLatency(Instant.now(), "test", null, false, null, 0, null);
+      latencyTracker.recordClientLatency(Instant.now(), "test", "test", false, null, 0, null);
+      latencyTracker.recordClientLatency(Instant.now(), "test", "test", false, Instant.now(), 0, null);
+      latencyTracker.recordClientLatency(Instant.now(), "test", "test", false, Instant.now(), TEST_AGGREGATE_LATENCY, httpOperation);
+
+      latencyTracker.recordClientLatency(null, null, null, null, false, null, 0, null);
+      latencyTracker.recordClientLatency(Instant.now(), null, null, null, false, null, 0, null);
+      latencyTracker.recordClientLatency(Instant.now(), Instant.now(), null, null, false, null, 0, null);
+      latencyTracker.recordClientLatency(Instant.now(), Instant.now(), "test", null, false, null, 0, null);
+      latencyTracker.recordClientLatency(Instant.now(), Instant.now(), "test", "test", false, null, 0, null);
+      latencyTracker.recordClientLatency(Instant.now(), Instant.now(), "test", "test", false, Instant.now(), 0, null);
+      latencyTracker.recordClientLatency(Instant.now(), Instant.now(), "test", "test", false, Instant.now(), TEST_AGGREGATE_LATENCY, httpOperation);
+
+      latencyTracker.recordClientLatency(testInstant, Instant.now(), null, null, false, null, 0, null);
+      latencyTracker.recordClientLatency(Instant.MAX, Instant.now(), null, null, false, null, 0, null);
+      latencyTracker.recordClientLatency(Instant.now(), Instant.MIN, null, null, false, null, 0, null);
+
+
+    } catch (Exception e) {
```

Comment:
remove the try/catch and rely on failures to go up to the test runner

Reply:
Removed the try-catch.